src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseExportImportResource.java

```diff
@@ -41,17 +41,15 @@
 import de.tum.in.www1.artemis.repository.hestia.ProgrammingExerciseTaskRepository;
 import de.tum.in.www1.artemis.security.Role;
 import de.tum.in.www1.artemis.service.AuthorizationCheckService;
+import de.tum.in.www1.artemis.service.ConsistencyCheckService;
 import de.tum.in.www1.artemis.service.CourseService;
 import de.tum.in.www1.artemis.service.SubmissionPolicyService;
 import de.tum.in.www1.artemis.service.exam.ExamAccessService;
 import de.tum.in.www1.artemis.service.feature.Feature;
 import de.tum.in.www1.artemis.service.feature.FeatureToggle;
 import de.tum.in.www1.artemis.service.programming.*;
 import de.tum.in.www1.artemis.web.rest.dto.RepositoryExportOptionsDTO;
-import de.tum.in.www1.artemis.web.rest.errors.AccessForbiddenException;
-import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
-import de.tum.in.www1.artemis.web.rest.errors.EntityNotFoundException;
-import de.tum.in.www1.artemis.web.rest.errors.InternalServerErrorException;
+import de.tum.in.www1.artemis.web.rest.errors.*;
```

Comment:
Code alert

Reply:
I think this is fine here. IntelliJ automatically does this, since we use a lot of classes from that package. So it would likely be a constant effort to not use * here.
In this file we also have * imports for some other packages.

Reply:
We might want to consider using the "Flag as Tolerated" functionality in Teamscale, then.
At least assuming that this will prevent it from being flagged in the future.

Reply:
We once decided that * imports are used once 4+ classes of the same package are imported.

Reply:
can we configure IntelliJ to follow this rule?

Reply:
Yes we can by adding this to the `.editorconfig` file:
```
[*.{java,kt}]
ij_java_class_count_to_use_import_on_demand = 4
```